wp-cache-sanitizer.php

```diff
@@ -0,0 +1,726 @@
+<?php
+
+/**
+* A sanitizer for WP Super Cache settings
+*
+* This class handles sanitizing the WP Super Cache settings. Each settings has a specific
+* value that can be evaluated before being inserted into the database or written to the wp-cache-config.php file.
+* This class is developed for the import/export tab in order to sanitize and sanitize the imported values.
+*
+* The settings are organized in terms of their allowed values: booleans, integers, strings, database options.
+*
+* Custom settings can be added by using the filters 'wp_super_cache_extend_allowed_{type}' to add the setting to the
+* allowed values and 'wp_super_cache_sanitize_{value}' to sanitize the new value. The value can then be sanitized using
+* this class's sanitize function.
+*
+* @package  WP Super Cache
+* @since 1.4.4
+*/
+
+class WP_Super_Cache_Sanitizer {
+
+  // The list of allowed database options
+  // Note: The stats and counts are commented out as they may not be necessary to import since they are site specific.
```

Comment:
We should make a decision.  If we're not going to import these, do we need to export them?